From f622f8eb9c1562178065f3cd2f3fd6feaa4e3f69 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Wed, 5 May 2021 13:30:14 +0200
Subject: [PATCH] Stop forcing static builds

CMakeLists.txt actually has some checks to make sure some configuration
options are valid in case of static or shared builds, but that's
completely useless as it then builds statically no matter what the user
has specified.
When not hardcoding static, the library can be build with shared libs or
without, depending on what value the user passes to -DBUILD_SHARED_LIBS
---
 CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c6262f6..09036cf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,7 +52,6 @@ if(WITH_OPENMP)
 endif()
 
 add_library(Library-C++
-    STATIC
     src/cluster.cpp
     src/direct.cpp
     src/direct_tree.cpp
-- 
2.31.1

